extensions/copymoveto/src/destination-form.tsx

```diff
@@ -1,10 +1,23 @@
-import { Action, ActionPanel, Form, LocalStorage, Toast, popToRoot, showToast } from "@raycast/api";
+import {
+  Action,
+  ActionPanel,
+  Form,
+  LocalStorage,
+  Toast,
+  popToRoot,
+  showToast,
+  getPreferenceValues,
+} from "@raycast/api";
 
 import { FormValidation, useForm } from "@raycast/utils";
 import { useEffect, useState } from "react";
 import { type Destination, destinationRepo } from "./repo/destination";
 import { checkDirExists } from "./utils/filesystem";
 
+interface Preferences {
+  showHiddenFolders: boolean;
+}
+
```

Comment:
```suggestion
```

You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`